VideoLocker/src/main/java/org/edx/mobile/user/UserAPI.java

```diff
@@ -135,4 +152,15 @@ String getUserEnrolledCoursesURL(@NonNull String username) {
         }
         return ret;
     }
+
+    @Nullable
+    private PrefManager getCurrentUserPrefs(String username) {
+        PrefManager prefManager = new PrefManager(MainApplication.instance(), PrefManager.Pref.LOGIN);
```

Comment:
rather than reference the pref file directly, could you make this a method in `LoginPrefs`?

Perhaps:
```java
class LoginPrefs {
  void setProfileImage(@Nullable ProfileImage profileImage);
}
```